Organization/Organization/AppDelegate.m

```diff
@@ -7,9 +7,92 @@ @interface AppDelegate ()
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions 
+@synthesize managedObjectModel;
+@synthesize managedObjectContext;
+@synthesize persistentStoreCoordinator;
+
+static NSManagedObjectContext *context;
```

Comment:
Very bad. Don't use static variables. If you need to access AppDelegate's instance - use [UIApplication sharedApplication].delegate